pyproject.toml

```diff
@@ -1,70 +1,109 @@
 [build-system]
-requires = ["setuptools>=45", "setuptools_scm[toml]>=6.2"]
+requires = ["setuptools>=69", "setuptools_scm[toml]>=8.0.4"]
 build-backend = "setuptools.build_meta"
 
 [project]
 name = "niftyone"
 description = "Large-scale neuroimaging visualization using FiftyOne"
-authors = [
-    {name = "Connor Lane", email = "connor.lane858@gmail.com"},
-]
+authors = [{name = "Connor Lane", email = "connor.lane858@gmail.com"}]
 readme = "README.md"
-requires-python = ">=3.7"
+requires-python = ">=3.7, <3.13"
```

Comment:
Should the minimum python version be bumped here as well to 3.8?

Reply:
Ahh, you're right - Missed this when I was going back and forth between debating whether or not to drop 3.7 support.